knowledge-portal/central/initiate-push-to-source.sh

```diff
@@ -6,15 +6,21 @@ set -e
 git config --global user.email "PlatformTeam@detecttechnologies.com"
 git config --global user.name "Detect Gitlab Bot"
 
-# changed files can be obtained by git diff between latest commit and unstaged area
-git reset --soft HEAD~1
-changed_files=$(git diff --name-only HEAD)
+# Install git lfs
+apt-get update -qq && apt-get install -y -qq git-lfs
+git lfs install
+git lfs track "*.jpg" "*.jpeg" "*.png" "*.gif" "*.svg" "*.ico" "*.mp3" "*.mp4"
+
+# changed files are all files since pipeline successfully ran last time on central
+
+GITLAB_API="https://gitlab.com/api/v4"
+PIPELINES=$(curl --header "PRIVATE-TOKEN: $PIPELINE_ID_TOKEN" "$GITLAB_API/projects/$CI_PROJECT_ID/pipelines?scope=finished&status=success" | jq '.[0].sha')
+LAST_PIPELINE_COMMIT=$(echo $PIPELINES | tr -d '"')
+changed_files=$(git diff --name-only $LAST_PIPELINE_COMMIT $CI_COMMIT_SHA)
```

Comment:
Please add this token generation part to the Usage section and possibly the other tokens/keys that have to be maintained.

Reply:
modified README accordingly